lib/docker-delta.coffee

```diff
@@ -11,7 +11,7 @@ Docker = require 'docker-toolbelt'
 
 docker = new Docker()
 
-DELTA_OUT_OF_SYNC_CODES = [23, 24]
+DELTA_OUT_OF_SYNC_CODES = [ 19, 23, 24 ]
```

Comment:
I don't see 19 in the [list of rsync exit codes](https://lxadm.com/Rsync_exit_codes) - have you seen it somewhere? or is there something special we do here that can cause it to exit with that code?

Reply:
Yeah, it's not listed anywhere I could find, but that's the status rsync exits when the src image is wrong (confirmed by reading its stderr).

Reply:
That is quite odd - may be a good idea to add a comment saying that you've seen that, in case someone in the future sees it and thinks it shouldn't be needed